sound/soc/intel/boards/sof_sdw.c

```diff
@@ -159,6 +159,22 @@ static const struct dmi_system_id sof_sdw_quirk_table[] = {
 					SOF_SDW_PCH_DMIC |
 					SOF_SDW_FOUR_SPK),
 	},
+	{
+		/*
+		 * this entry covers multiple HP skus. The family name
```

Comment:
I don't understand the comment. SKU is what is reported by dmidecode and how we track devices to avoid sharing code names or product name.

Reply:
> I don't understand the comment. SKU is what is reported by dmidecode and how we track devices to avoid sharing code names or product name.

I guess @lyakh meant "SKU" is an abbreviation, so it should be capital. ie. "sku" => "SKU" in the comment.

Reply:
oh, right. I thought it was about the spelling I missed the point on using capital letters. Thanks for the comment.

Reply:
fixed.